test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1210,6 +1210,43 @@ def forward(self, x):
                 for key in n.meta:
                     self.assertEqual(n.meta[key], weight_meta[key])
 
+    def test_xnnpack_module_name_with_underscores(self) -> None:
```

Comment:
can you move this to [test_xnnpack_quantizer.py](https://github.com/pytorch/pytorch/blob/main/test/quantization/pt2e/test_xnnpack_quantizer.py)?

Reply:
Moved to that file